platform/web/doc_classes/EditorExportPlatformWeb.xml

```diff
@@ -86,6 +86,12 @@
 			If [code]true[/code], the exported game will support threads. It requires [url=https://web.dev/articles/coop-coep]a "cross-origin isolated" website[/url], which may be difficult to set up and is limited for security reasons (such as not being able to communicate with third-party websites).
 			If [code]false[/code], the exported game will not support threads. As a result, it is more prone to performance and audio issues, but will only require to be run on an HTTPS website.
 		</member>
+		<member name="variant/emscripten_pool_size" type="int" setter="" getter="">
+			TODO
```

Comment:
Something like "Number of threads that emscripten will allocate at startup. A smaller value will allocate fewer threads and consume fewer system resources, but you may run the risk of running out of threads in the pool and needing to allocate more threads at run time which may cause a deadlock. [b]Note:[/b] some browsers have a hard cap on the number of threads that can be allocated, so it is best to be cautious and keep this number low."

CC @Faless

Reply:
This sounds great